cmd/kubelet/app/options/options.go

```diff
@@ -75,7 +75,7 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	// DEPRECATED: Remove these flags at the beginning of 1.5.
 	fs.Var(&s.AuthPath, "auth-path", "Path to .kubernetes_auth file, specifying how to authenticate to API server.")
 	fs.MarkDeprecated("auth-path", "will be removed in a future version")
-	fs.StringSliceVar(&s.APIServerList, "api-servers", []string{}, "List of Kubernetes API servers for publishing events, and reading pods and services. (ip:port), comma separated.")
+	fs.StringSliceVar(&s.APIServerList, "api-servers", []string{}, "List of Kubernetes API servers for publishing events, and reading pods and services. (host:port (port is optional)), comma separated.")
```

Comment:
Can drop the parenthesis:

`List of Kubernetes API servers for publishing events and reading pods and services. Comma separated list of hostnames or host:port pairs.`